app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :yard_sales
+  has_one :saved_yardsale
+
```

Comment:
Se need to specify that the user is a tagger with act_as_tagger

Reply:
has been updated